packages/app/src/index.js

```diff
@@ -3,11 +3,20 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import NominationShow from './components/nominationShow';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <Router>
+    <div className='container'>
+      <React.StrictMode>
+        <Switch>
+          <Route path="/" exact component={App} />
+          <Route path="/nomination/:id" exact component={NominationShow} />
+        </Switch>
+      </React.StrictMode>
+    </div>
+  </Router>,
```

Comment:
You're importing but not using App.js now. Maybe it would make more sense to put the router *in* App.js?

Reply:
I asked Dan about this, and he said that the App component right now just represents a button and probably won't be used, so it didn't make sense to move the router into that component.